Content.Server/Salvage/SalvageMapPrototype.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Content.Server.Objectives.Interfaces;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.Salvage
+{
+    [Prototype("salvageMap")]
+    public class SalvageMapPrototype : IPrototype
+    {
+        [ViewVariables]
+        [DataField("id", required: true)]
+        public string ID { get; } = default!;
+
+        /// <summary>
+        /// Relative directory path to the given map, i.e. `Maps/Salvage/test.yml`
+        /// </summary>
+        [ViewVariables]
+        [DataField("mapPath", required: true)]
+        public string MapPath { get; } = default!;
+
+        /// <summary>
+        /// Size *from 0,0* in units of the map (used to determine if it fits)
+        /// </summary>
+        [ViewVariables]
+        [DataField("size", required: true)]
+        public float Size { get; } = 1.0f;
```

Comment:
This is kinda uggo but the only other solution I can think of rn is spawning it significantly out, grabbing LocalBounds, then moving it.

Reply:
make spawning grids in nullspace & moving em after possible whenst

Reply:
No idea how to do this, can update it some day once the engine is able to help do it.